desktop/src/main/java/bisq/desktop/main/offer/MutableOfferDataModel.java

```diff
@@ -121,10 +120,13 @@
     protected final ObjectProperty<Price> price = new SimpleObjectProperty<>();
     protected final ObjectProperty<Volume> volume = new SimpleObjectProperty<>();
     protected final ObjectProperty<Volume> minVolume = new SimpleObjectProperty<>();
+    protected final CoinFormatter btcFormatter;
 
     // Percentage value of buyer security deposit. E.g. 0.01 means 1% of trade amount
     protected final DoubleProperty buyerSecurityDeposit = new SimpleDoubleProperty();
 
+    protected final StringProperty buyerSecurityDepositInBTC = new SimpleStringProperty();
```

Comment:
This property is set, but never used. How can this fix the issue with the security deposit?

Reply:
`buyerSecurityDepositInBTC` is already used by a listener in MutableOfferView.java.  Its odd that github diff is showing the line above as "new code", as I think @BtcContributor only added the keyword `protected` here.

Reply:
NACK. -> upon testing this PR it does not fix the case of duplicate on offers that had 0.006 security deposit.